### PR TITLE
Default message to select time

### PR DESCRIPTION
**Pull request by joelhmetz**
Added default message for language translation

Image: https://uploaddeimagens.com.br/imagens/xo9X4Us

### CODE REVIEW

**Comment by REJack** [comment]
add this `Zeit auswählen` as german translation (de) 😄

**Comment by joelhmetz** [comment]
> add this `Zeit auswählen` as german translation (de) 😄

add

**Comment by joelhmetz** [comment]
> add this `Zeit auswählen` as german translation (de) 😄

I will wait for approval first, after approved, I add the requested.

File: resources/views/components/time-picker.blade.php [hunk 1]
```diff
@@ -102,6 +102,7 @@ class="w-4 h-4 text-secondary-400 hover:text-negative-400 transition-all ease-ou
                     :component="WireUiComponent::resolve('input')"
                     id="search.{{ $attributes->wire('model')->value() }}"
                     label="Select time"
```
Comment (ph7jack) [inline resources/views/components/time-picker.blade.php:104]: remove the old label prop

File: src/lang/de/messages.php [hunk 2]
```diff
@@ -1,16 +0,0 @@
-<?php
```
Comment (ph7jack) [inline src/lang/de/messages.php]: Why this translation was deleted?

File: src/lang/pt_BR/messages.php [hunk 3]
```diff
@@ -2,6 +2,7 @@
 <?php
 
 return [
+    'selectTime' => 'Selecione o tempo',
```
Comment (ph7jack) [inline src/lang/pt_BR/messages.php]:
```suggestion
    'selectTime' => 'Selecione a hora',
```

**Review by ph7jack — CHANGES_REQUESTED** [review]
@joelhmetz 
First, my apologizes for my long time off, then thanks for your contribution

**Comment by ph7jack** [comment]
After fix these comments I'll approve this PR, thanks

File: src/lang/de/messages.php (hunk 2 above)
Comment (joelhmetz) [reply src/lang/de/messages.php]: When I made the first version it passed the tests, already with the language more, it was denied.
I'll wait for approval, after I launch the translation.

Comment (ph7jack) [reply src/lang/de/messages.php]: Hummm

**Comment by ph7jack** [comment]
Good, please, run the phpcs fixer on these files:
- src/lang/en/messages.php
- src/lang/es/messages.php
- src/lang/pt_BR/messages.php

And rollback this file `src/lang/de/messages.php`

After changes it, I will approve and merge

**Comment by joelhmetz** [comment]
Sorry for the mess, I'm new to contributing...

**Review by ph7jack — APPROVED** [review]
Thanks @joelhmetz

**Pull request merged**